odoo/addons/base/tests/test_translate.py

```diff
@@ -737,7 +737,7 @@ def test_sync_update(self):
         self.assertEqual(len(translations), 2)
 
         # modifying the arch should sync existing translations without errors
-        new_arch = archf % ('Subtotal', 'Subtotal:<br/>')
+        new_arch = archf % ('Subtotal', 'Subtotal : <br/>')
```

Comment:
Please explain this change.

Reply:
Before my change, the difference between `Subtotal` and `Subtotal:<br/>` was big enough to be flagged by the `difflib.get_close_matches()` method ( with the sensitivity we have; 0.9 ).

Since my change, we do not take xml/html tags into account anymore.
So the difference between `Subtotal` and `Subtotal:` is not enough.
So I just added textual changes in the test "new translation" to pass the 0.9 threshold.

So it's just a "trick" to keep the test working as intended.